### DISCUSSION TITLE
After unchecking all filters in explore, when reactivating them the app is ignoring the change.

### DISCUSSION THREAD

**Discussion by thomaswitt** [community/community#47043] [Mobile] [answered]
In the explore tab:
- Uncheck all filters; save
- recheck one or all filters; save
- nothing is displayed; when opening filter again, all radio buttons are unchecked 

<details>
GitHub for iOS, Version 1.99.0 (90728769)
Device: iPhone15,3 (iOS 16.3)
</details>

**Comment by mattruggio** [comment, marked as answer]
Hey @thomaswitt thanks for the community discussion submission!  I just verified the issue you see was reproducible in 1.99.0 and was fixed as of 1.100.0.  Expect this to be fixed once 1.100.0 hits Apple's App Store (usually updates about once a week).
